DTS-STN/canadian-dental-care-plan
# Adding/updating lookup identifier for countries

### Description
- Renaming `COUNTRY_CODE_CANADA` to `CANADA_COUNTRY_ID` and `COUNTRY_CODE_USA` to `USA_COUNTRY_ID` and removing the default value for these variables
- Fixing inconsistencies between `.env.example` and `env.server`
- Fixing some broken parts of `/federal-provincial-territorial-benefits` and `/personal-information` within the `/apply` flow that https://github.com/DTS-STN/canadian-dental-care-plan/pull/681 broke

### Checklist

### Test Instructions
Copy the updated `.env.example` into your local `.env` file and run the application locally. Start the apply flow and navigate to `/federal-provincial-territorial-benefits`. The page should render correctly
